analyzers/src/SonarAnalyzer.Common/Helpers/SonarLintXmlReader.cs

```diff
@@ -98,4 +94,6 @@ private static SonarLintXml ParseContent(SourceText sonarLintXml)
 
     private static bool ReadBoolean(string str, bool defaultValue = false) =>
         bool.TryParse(str, out var propertyValue) ? propertyValue : defaultValue;
+
+    private string LanguageXmlName(string language) => language.Equals(LanguageNames.CSharp) ? "cs" : "vbnet";
```

Comment:
This expression body is not trivial, should be on the next line

Reply:
I don't see a reason to use `Equals` here. That looks like Java code

Reply:
This should be a switch expression that throws for unknown languages. As we explicitly register only for C# and VB and never ever should we be called for any other.